containers/support/AttachScreenshot.js

```diff
@@ -0,0 +1,63 @@
+import React, { useState } from 'react';
+import { c } from 'ttag';
+import { Text, Icon, Button, FileInput, useNotifications } from 'react-components';
+import { toBlob, resize } from 'proton-shared/lib/helpers/image';
+import { MAX_SIZE_SCREENSHOT } from 'proton-shared/lib/constants';
+import PropTypes from 'prop-types';
+
+const AttachScreenshot = ({ id, onUpload, onReset }) => {
+    const [attached, setAttached] = useState(false);
+    const { createNotification } = useNotifications();
+
+    const handleClick = () => {
+        setAttached(false);
+        onReset();
+    };
+
+    const handleChange = async ({ target }) => {
+        const images = [...target.files].filter(({ type }) => /^image\//i.test(type));
+
+        if (!images.length) {
+            return createNotification({
+                type: 'error',
+                text: c('Error notification in the bug report modal when the user upload file').t`No image selected`
+            });
+        }
+
+        setAttached(true);
+        onUpload(
+            await Promise.all(
```

Comment:
it should be a helper.

Reply:
I didn't get this one. It's a `Promise.all` running a helper function and formatting a specific result. Also, the logic is already extracted inside a helper.